scripts/bootstrap.sh.in

```diff
@@ -136,6 +136,10 @@ populate_registry() {
         ${SALT_CALL} --retcode-passthrough state.apply metalk8s.registry.populated saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
 }
 
+run_etcd() {
+        ${SALT_CALL} metalk8s.kubeadm.init.etcd saltenv=metalk8s-@VERSION@ pillarenv=metalk8s-@VERSION@
```

Comment:
Missing `--retcode-passthrough` and `state.apply`?

Reply:
Yep, it's fixed